tasks/version_shell.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if test -f /opt/puppetlabs/puppet/VERSION; then
+  cat /opt/puppetlabs/puppet/VERSION
```

Comment:
If we're writing a task in a supported module I think it should return JSON

Reply:
Do you think it needs to be extensible, or some other reason? And what info specifically about how we found it?

Reply:
Made it `{"version": "<ver>", "source": "/opt/puppetlabs/puppet/VERSION"}`.